service/src/main/kotlin/fi/espoo/evaka/placement/PlacementService.kt

```diff
@@ -155,54 +145,53 @@ private fun clearOldPlacements(h: Handle, childId: UUID, from: LocalDate, to: Lo
                 // old placement is within new placement (or identical)
                 !old.startDate.isBefore(from) && !old.endDate.isAfter(to) -> {
                     checkAclAuth(aclAuth, old)
-                    h.cancelPlacement(old.id)
+                    val (_, _, _) = handle.cancelPlacement(old.id)
```

Comment:
is `val (_, _, _) = ` necessary?

Reply:
It's not the only option, but all branches of `when` need to return the same type `Unit`.
`val _ = ...` doesn't work, because Kotlin has a bit strange rules about where underscores can be used.